src/global/project/parsed_manifest.rs

```diff
@@ -3,21 +3,52 @@ use std::str::FromStr;
 
 use indexmap::{IndexMap, IndexSet};
 use itertools::Itertools;
+use miette::IntoDiagnostic;
 use pixi_manifest::PrioritizedChannel;
 use rattler_conda_types::{NamedChannelOrUrl, PackageName, Platform};
 use serde::de::{Deserialize, DeserializeSeed, Deserializer, MapAccess, Visitor};
+use serde::Serialize;
 use serde_with::{serde_as, serde_derive::Deserialize};
 
 use super::environment::EnvironmentName;
 
 use super::error::ManifestError;
+use super::ExposedData;
 use pixi_spec::PixiSpec;
 
 /// Describes the contents of a parsed global project manifest.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize)]
 pub struct ParsedManifest {
     /// The environments the project can create.
-    environments: IndexMap<EnvironmentName, ParsedEnvironment>,
+    envs: IndexMap<EnvironmentName, ParsedEnvironment>,
+}
+
+impl<I> From<I> for ParsedManifest
+where
+    I: IntoIterator<Item = ExposedData>,
+{
+    fn from(value: I) -> Self {
+        let mut envs: IndexMap<EnvironmentName, ParsedEnvironment> = IndexMap::new();
+        for data in value {
+            let ExposedData {
+                env,
+                platform,
+                channel,
+                package,
+                binary,
+                exposed,
+            } = data;
+            let mut parsed_environment = envs.entry(env).or_default();
+            parsed_environment.channels.insert(channel);
+            parsed_environment.platform = platform;
+            parsed_environment
+                .dependencies
+                .insert(package, PixiSpec::default());
```

Comment:
I was considering that, but how would the matchspec then look like?

Most users will run `pixi global install ipython` rather than `pixi global install ipython=8.27`, but both will result in version `8.27.0`.

I would argue in the first case you want `*`or maybe `>8.27`. But in the second case, you want `>8.27,<9.0`.
I was hoping by using `*` everywhere I would match the most common use case.

Reply:
We agreed, on asking the user during the interactive prompt.
They will get the choice between pinning to an exact version `=8.27.0` or use a glob `*`.
I propose to make the default answer `*` which will also be used by `--assume-yes`.

We also agree that no matter the choice the user makes the migration shouldn´t lead to different versions. Maybe our caching approach will be enough to take care of that.

Reply:
It wasn't super obvious to implement, and I'd rather not block on this. I've opened an issue on our notion board for now